src/calculateRentalCost.js

```diff
@@ -4,7 +4,22 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const oneDayRent = 40;
```

Comment:
Prefer using `const` over `let` for `sumForRent` since its value is not being reassigned after its initial calculation. This ensures the variable is not changed unintentionally later in the code, in accordance with checklist item 3.